docs/capabilities/dispenser-client.md

```diff
@@ -0,0 +1,57 @@
+# TestNet Dispenser Client
+
+The TestNet Dispenser Client is a utility for interacting with the AlgoKit TestNet Dispenser API. It provides methods to fund an account, register a refund for a transaction, and get the current limit for an account.
+
+## Creating a Dispenser Client
+
+To create a Dispenser Client, you need to provide an authorization token. This can be done in two ways:
+
+1. Pass the token directly to the client constructor as `authToken`.
+2. Set the token as an environment variable `ALGOKIT_DISPENSER_ACCESS_TOKEN` (see [docs](https://github.com/algorandfoundation/algokit/blob/main/docs/testnet_api.md#error-handling) on how to obtain the token).
+
+If both methods are used, the constructor argument takes precedence.
+
+```ts
+import * as algokit from '@algorandfoundation/algokit-utils'
+
+// Using constructor argument
+const client = new algokit.getDispenserApiTestnetClient({ authToken: 'your_auth_token' })
+
+// Using environment variable
+process.env['ALGOKIT_DISPENSER_ACCESS_TOKEN'] = 'your_auth_token'
+const client = new DispenserApiTestnetClient()
```

Comment:
Should this also use the algokit method rather than the client class directly? And if so should that method be explicit it's using environment in the method name like some of the others?

Reply:
was just a typo, the constructor within the client by default will throw and error if there is no env and no token supplied so not sure we need a separate method just for env loading since user can just init without passing anything (while making sure env is available, and thats also documented in detail in docstrings)